components/delinquency_modal/useDelinquencyModalController.tsx

```diff
@@ -45,7 +45,7 @@ export const useDelinquencyModalController = (props: UseDelinquencyModalControll
     };
 
     useEffect(() => {
-        if (delinquencyModalPreferencesConfirmed.length === 0 && product == null && !requestedProducts) {
+        if (delinquencyModalPreferencesConfirmed.length === 0 && product === undefined && !requestedProducts && isCloud) {
```

Comment:
Was this mounted even before accessing the modal? Just want to make sure this solves the problem that was reported in community

Reply:
Yes, the modal is mounted in root